docs/usage/configuration-options.md

```diff
@@ -1763,6 +1763,23 @@ Example config:
 }
 ```
 
+### maxRetryAfter
+
+By default, Renovate will retry after any `Retry-After` header value, up to a maximum of 60 seconds.
```

Comment:
```suggestion
At times, remote hosts will return a 4xx response with a `Retry-After` header value, indicating that Renovate has been rate limited but may retry after a certain waiting period.
By default, Renovate will retry after any `Retry-After` header value, up to a maximum of 60 seconds.
If the `Retry-After` value is higher than 60 seconds, Renovate will instead abort the job.
```

Reply:
```suggestion
A remote host may return a `4xx` response with a `Retry-After` header value, which indicates that Renovate has been rate-limited.
Renovate may try to contact the host again after waiting a certain time, that's set by the host.
By default, Renovate tries again after the `Retry-After` header value has passed, up to a maximum of 60 seconds.
If the `Retry-After` value is more than 60 seconds, Renovate will abort the job instead of waiting.
```

How about something like this?